### DISCUSSION TITLE
疑问：请问挖满了的磁盘，如何处理？

### DISCUSSION THREAD

**Discussion by lixbai** [Chia-Network/chia-blockchain#5370] [Support]
各位好：
我有一个2T的机械硬盘，用来存储plot 文件，挖了几天，快要满了。
但是我一个块都没有获得。
请问我应该怎么处理这些机械硬盘里面的plot 文件？

或者说chia 挖矿，这个文件存储的机制，是怎么样的？
难道要一直存储下去吗？
这样的话，我估计到最后，随着人数，机器达到饱和，收益越来越少，但是手里的硬盘越来越多，到时候谁都很难承担购买硬盘的开支啊？？？
换个极端的说法，如果不能删除，随着chia挖矿的进行。有可能会把世界上的硬盘都填满？？？这怎么可能，难道chia的开发人员没有想过这个问题吗？

这个是我目前的认知，不知道应该怎么处理这些挖出来的plot文件？

期待各位的探讨，
谢谢。

**Comment by nuistzj2014** [comment]
黄花菜项目，你还想咋地

**Reply by lixbai** [reply]
请问应该怎么处理？

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
